learn/platform/links.md

```diff
@@ -5,7 +5,7 @@ description: If you're new to Moonbeam or the Polkadot network, here are some im
 
 # Links
 
- - **[Substrate Docs](https://docs.substrate.io){target=\_blank}** - starting point for learning about [Substrate](/learn/platform/glossary/#substrate){target=\_blank}, a Rust-based framework for developing blockchains. Moonbeam is developed using Substrate and uses many of the modules that come with it
+ - **[Polkadot Docs](https://docs.polkadot.com/){target=\_blank}** - starting point for learning about the [Polkadot SDK](/learn/platform/glossary/#substrate){target=\_blank}, a Rust-based framework for developing blockchains. Moonbeam is developed using Substrate and uses many of the modules that come with it
```

Comment:
This might be confusing for users who click on the Polkadot SDK link and then they get taken to the glossary for Substrate. Do we have an entry for Polkadot SDK?

Reply:
If not, maybe just link directly to here: https://docs.polkadot.com/develop/parachains/intro-polkadot-sdk/